Documentation/configuration.md

```diff
@@ -1,70 +1,60 @@
-files there (e.g. documentation).
-
-Every configuration file has two common fields: `rktKind` and
-`rktVersion`. Both are strings, and the rest of the fields are specified by
-that pair. The currently supported kinds and versions are described below.
+`rkt` reads configuration from two directories - a **system directory** and a **local directory**.
+The system directory defaults to `/usr/lib/rkt` and the local directory `/etc/rkt`, but both can be overridden via command-line flags.
+
+`rkt` looks for configuration files inside subdirectories of these two directories.
+It ignores everything but regular files with `.json` extension.
+This means `rkt` does _not_ search for the files by recursively going down the directory tree.
+This also means that users are free to put some additional files there (e.g. documentation).
+
+Every configuration file has two common fields: `rktKind` and `rktVersion`.
+Both are strings, and the rest of the fields are specified by that pair.
+The currently supported kinds and versions are described below.
 These fields must be specified and cannot be empty.
 
-`rktKind` describes the type of the configuration. This is to avoid putting
-unrelated values into single monolithic file.
+`rktKind` describes the type of the configuration.
+This is to avoid putting unrelated values into single monolithic file.
 
-`rktVersion` allows configuration versioning for each kind of configuration. A
-new version should be introduced when doing some backward-incompatible changes:
-for example, when removing a field or incompatibly changing its semantics. When
-a new field is added, a default value should be specified for it, documented,
-and used when the field is absent in file. This way, an older version of `rkt`
-can work with newer-but-compatible versions of configuration files, and newer
-versions of `rkt` can still work with older versions of configuration files.
+`rktVersion` allows configuration versioning for each kind of configuration.
+A new version should be introduced when doing some backward-incompatible changes: for example, when removing a field or incompatibly changing its semantics.
+When a new field is added, a default value should be specified for it, documented,
+and used when the field is absent in file.
+This way, an older version of `rkt` can work with newer-but-compatible versions of configuration files, and newer versions of `rkt` can still work with older versions of configuration files.
 
-The configuration in the system directory can be overridden by configuration
-in the local directory. The semantics of configuration override are specific to
-the kind and version of the configuration file and are described below.
+The configuration in the system directory can be overridden by configuration in the local directory.
+The semantics of configuration override are specific to the kind and version of the configuration file and are described below.
 Filenames do not play any role in overriding.
```

Comment:
File names are not examined to determine local overrides. Only the fields inside configuration files need to match.